docs/docs/frigate/installation.md

```diff
@@ -108,10 +108,10 @@ The Hailo-8L is an M.2 card typically connected to a carrier board for PCIe, whi
 
 For Raspberry Pi 5 users with the AI Kit, installation is straightforward. Simply follow this [guide](https://www.raspberrypi.com/documentation/accessories/ai-kit.html#ai-kit-installation) to install the driver and software.
 
-For other boards, follow these steps for installation:
+For other installations, follow these steps for installation:
 
 1. Install the driver from the [Hailo GitHub repository](https://github.com/hailo-ai/hailort-drivers). A convenient script for Linux is available to clone the repository, build the driver, and install it.
-2. Copy or download [this script](https://gist.github.com/spanner3003/4b85751d671d4ac55f926e564f1abc3e#file-install_hailo8l_driver-sh).
+2. Copy or download [this script](https://github.com/blakeblackshear/frigate/blob/41c9b13d2fffce508b32dfc971fa529b49295fbd/docker/hailo8l/user_installation.sh).
 3. Ensure it has execution permissions with `sudo chmod +x install_hailo8l_driver.sh`
```

Comment:
The file name has changed to `user_installation.sh`

Reply:
right, the link in the docs is correct https://deploy-preview-13787--frigate-docs.netlify.app/frigate/installation#installation

Reply:
I think he means it also needs to be updated in step 3 as well.

Reply:
fixed in latest PR